src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -153,7 +170,7 @@ const LoginForm: React.FunctionComponent<PropsType> = ({
           )}
         </div>
 
-        {venue.requiresTicketCode && (
+        {venue.access?.includes(VenueAccessMode.Codes) && (
```

Comment:
As above RE: `venue.access` seemingly needing to be an array (yet we changed the type of it I think)